assembly/battle_scripts/ability_battle_scripts.s

```diff
@@ -2,7 +2,7 @@
 .thumb
 .align 2
 
-.include "..\\defines"
```

Comment:
this will not work on linux. I changed all of this kind of double back slashes to single slash (why the hell double back slashes are used tho? )